systemd/coreos-installer-generator

```diff
@@ -10,7 +10,7 @@ exec 1>/dev/kmsg; exec 2>&1
 
 UNIT_DIR="${1:-/tmp}"
 
-cmdline=( $(</proc/cmdline) )
+IFS=" " read -r -a cmdline <<< "$(</proc/cmdline)"
```

Comment:
Couldn't this just be:

```suggestion
IFS=" " read -r -a cmdline < /proc/cmdline
```

Reply:
I don't know. I copy / pasted the fix from https://github.com/coreos/coreos-installer/pull/999/files#diff-a31b15f1dd66afbb6fe505b362441e553a822b80292f285a5ef93cb89cb0b233R21

Reply:
While I think we could probably clean this up, I agree that there's value to staying consistent.